ikasaneip/developer/mvn-archetype/ikasan-standalone-db-jms-im/pom.xml

```diff
@@ -53,15 +53,19 @@
 
     <properties>
         <maven.javadoc.skip>true</maven.javadoc.skip>
-        <maven.compiler.source>17</maven.compiler.source>
-        <version.org.springboot>2.3.4.RELEASE</version.org.springboot>
-        <maven.compiler.target>17</maven.compiler.target>
-        <version.com.h2database>1.4.200</version.com.h2database>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <version.ikasan>3.2.2</version.ikasan>
-        <version.activemq>5.16.3</version.activemq>
     </properties>
 
+<!--    <properties>-->
```

Comment:
why all properties are commented

Reply:
those properties are not relevant at that level and in fact require the developer to maintain them. All are derived from parent level poms.